content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -883,8 +911,14 @@ private String getSubmissionId(ContentReviewItem item, String fileName, Site sit
 			// Build header maps
 			Map<String, Object> data = new HashMap<String, Object>();
 			data.put("owner", userID);
+			if(site != null) {
+				data.put("owner_default_permission_set", mapUserRole(userID, site.getId(), false));
+			}
 			if (StringUtils.isNotBlank(submitterID)) {
-				data.put("submitter", submitterID);	
+				data.put("submitter", submitterID);
+				if(site != null) {
+					data.put("submitter_default_permission_set", mapUserRole(submitterID, site.getId(), false));
```

Comment:
Can this be put into the above `if(site != null)` block? Conditional seems duplicated.